helm/aladdin-demo/values.yaml

```diff
@@ -29,38 +33,61 @@ apm:
   datadog_trace_agent_hostname: datadog-agent.kube-system
   datadog_env:
 
-deploy:
-  # number of seconds for the containers to perform a graceful shutdown, after which it is voilently terminated
+affinity: critical-datadog-apm
```

Comment:
added this default here, and disabled it for LOCAL

Reply:
don't love putting this here, since it's fivestars specific, and this is open sourced

Reply:
true - i just _moved_ it here, to centralize it since it was repeated in the other yaml files. In the same vein, *KDEV|KSTAG|KPROD* are also FS specific.
I guess we can add a note here and in the README to clarify that this configuration needs to be customized by the implementor.